domains/service/src/config.rs

```diff
@@ -147,7 +147,7 @@ impl From<SubstrateConfiguration> for Configuration {
                 max_blocks_per_request: 64,
                 // Substrate's default, full mode
                 sync_mode: SyncMode::Full,
-                pause_sync: Arc::new(AtomicBool::new(false)),
+                pause_sync: Arc::new(AtomicBool::new(true)),
```

Comment:
Just double-checking how this config change works? Do we un-pause sync somewhere else?

I had a look at the docs, but they don't really explain much:
https://github.com/autonomys/polkadot-sdk/blob/5626154d0781ac9a6ffd5a6207ed237f425ae631/substrate/client/network/src/config.rs#L632-L633 

Maybe a comment here would help?

Reply:
This parameter disables block requests for Substrate syncing engine. The change is required because I enabled the block request protocol and by default domains will start syncing blocks which is incorrect for domains because we create them only from local consensus blocks.